src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -57,20 +61,8 @@
     private static final int ANY_OTHER_EXCEPTION_EXIT_VALUE = 3;
     private static final String STACK_TRACE_ON_USER_EXCEPTION_PROPERTY = "GATK_STACKTRACE_ON_USER_EXCEPTION";
 
-    /**
-     * The packages we wish to include in our command line.
-     */
-    protected List<String> getPackageList() {
-        final List<String> packageList = new ArrayList<>();
-        packageList.addAll(Arrays.asList("org.broadinstitute.hellbender"));
-        return packageList;
-    }
-
-    /**
-     * The single classes we wish to include in our command line.
-     */
-    protected List<Class<? extends CommandLineProgram>> getClassList() {
-        return Collections.emptyList();
+    protected GATKConf getConfiguration() {
+        return GATKConfBuilder.getDefaultConfiguration();
```

Comment:
I agree partially on this. It is better to have `GATKConf` in `CommandLineProgram`, but if this method remains, to allow extending `Main` classes to set their own configuration on startup. This is important for one reason: if an API client wants its own configuration and it only lives in `CommandLineProgram` accessible by subclasses, it requires that every time a tool is implemented this method should be overridden. Thus, it makes more dirty to implement walker types which only requires an apply, for example.